### PR TITLE
MARBLE-1886 Refactor Card to DisplayCard

### PR DESCRIPTION
**Pull request by danthewolfe**
* Refactor Card component so it doesn't need quite so many weird workarounds.
* `DisplayCard` replaces `Card`
* `MarbleItemCard` replaces `ManifestCard`
* "Empty" card in `CardShell`
* `CardGroup` now accepts an optional array of breakpoint sizes (for adjusting the number of cards when the `CardGroup` is not full width.

See also:
https://github.com/ndlib/inquisition/pull/39
https://github.com/ndlib/marble-web/pull/106

### CODE REVIEW

**Review by jonhartzler — CHANGES_REQUESTED** [review]
I have a few comments that I left in line but mostly what I am wondering about is 2 things.   Since this is such an import section in our code can we get some small usage comments on the top of the components describing what they are used for and props and a few more tests.  We are jumping headfirst into a CD thing here and I am concerned we don't have enough protection for ourselves.   thoughts ?

File: @ndlib/gatsby-theme-marble/src/components/Shared/DisplayCard/CardBadge/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/** @jsx jsx */
```
[the remark this answers is not in the document]
Comment (danthewolfe) [reply @ndlib/gatsby-theme-marble/src/components/Shared/DisplayCard/CardBadge/index.js:1]: Added unit test.

File: @ndlib/gatsby-theme-marble/src/components/Shared/CardGroup/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ allowToggle - true to see the toggle false to not display
 defaultState - the default state the toggle should be in grid or list
```
[the remark this answers is not in the document]
Comment (danthewolfe) [reply @ndlib/gatsby-theme-marble/src/components/Shared/CardGroup/index.js]: corrected and updated.

File: @ndlib/gatsby-theme-marble/src/components/Shared/DisplayCard/CardBadge/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      )
+    case 'pdf':
+      return (
+        <span sx={containerSx}>
+          <IconContext.Provider value={{ color: 'white' }}>
+            <MdPictureAsPdf sx={iconSx}/>
+          </IconContext.Provider>
+        </span>
+      )
+    default:
```
[the remark this answers is not in the document]
Comment (danthewolfe) [reply @ndlib/gatsby-theme-marble/src/components/Shared/DisplayCard/CardBadge/index.js:57]: added

File: @ndlib/gatsby-theme-marble/src/components/Shared/DisplayCard/CardShell/LinkOrWrapper/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+const LinkOrWrapper = ({ target, children }) => {
+  const linkSx = {
+    display: 'block',
+    position: 'relative',
+  }
+  return target
+    ? (
+      <Link
+        to={target}
+        sx={{
```
[the remark this answers is not in the document]
Comment (danthewolfe) [reply @ndlib/gatsby-theme-marble/src/components/Shared/DisplayCard/CardShell/LinkOrWrapper/index.js]: I've moved all the styling to the card to a single `sx.js` file at the root of `DisplayCard`.

**Review by jonhartzler — APPROVED** [review]

**Pull request merged**